client/app/bundles/Home/home.js

```diff
@@ -2,8 +2,8 @@ import Tabslet from 'tabslet';
 import $ from 'jquery'
 require('../../assets/styles/home.scss')
 require('./bootstrap_carousel.js')
+require('./startup/clientRegistration.jsx')
```

Comment:
You need to make a tools bundle. The homepage was deliberately left light-weight to maximize fast load speeds